src/functions/module_logical.ts

```diff
@@ -49,27 +51,33 @@ export const IF = {
   description: _t("Returns value depending on logical expression."),
   args: [
     arg(
-      "logical_expression (boolean)",
+      "logical_expression (boolean, range<boolean>)",
       _t(
         "An expression or reference to a cell containing an expression that represents some logical value, i.e. TRUE or FALSE."
       )
     ),
-    arg("value_if_true (any)", _t("The value the function returns if logical_expression is TRUE.")),
     arg(
-      "value_if_false (any, default=FALSE)",
+      "value_if_true (any, range)",
```

Comment:
```suggestion
      "value_if_true (any, range<any>)",
```
probably a bit more clear no ? Otherwise I'm wondering why `range` is not included in `any`

Reply:
both notations are possible, we should clean everything up if we want only one representation

Reply:
if both are possible (atm) i think it doesn't hurt to add <any> to avoid misunderstandings